lib/rubocop/cop/style/conditional_assignment.rb

```diff
@@ -288,10 +293,6 @@ def assignment_node(node)
           assignment
         end
 
-        def condition?(node)
```

Comment:
It is unrelated to the fix. This was done to decrease the length of the class. Would you prefer that this change be removed all together and the increase the max class length to 172, or would you prefer that this change be placed into a different commit?